pom.xml

```diff
@@ -81,28 +64,76 @@
 
     </dependencies>
 
-    <build>
-        <plugins>
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-compiler-plugin</artifactId>
-                <version>3.7.0</version>
-                <configuration>
-                    <source>1.8</source>
-                    <target>1.8</target>
-                </configuration>
-            </plugin>
-        </plugins>
-    </build>
-
-    <repositories>
-        <repository>
-            <snapshots>
-                <enabled>false</enabled>
-            </snapshots>
-            <id>forgerock-private-releases</id>
-            <name>ForgeRock Private Release Repository</name>
-            <url>http://maven.forgerock.org/repo/private-releases</url>
-        </repository>
-    </repositories>
+	<repositories>
+	    <repository>
+	      <snapshots>
+		<enabled>false</enabled>
+	      </snapshots>
+	      <id>forgerock-internal-releases</id>
+	      <name>ForgeRock Release Repository</name>
+	      <url>http://maven.forgerock.org/repo/internal-releases</url>
+	    </repository>
+	    <repository>
+	      <snapshots>
+		<enabled>false</enabled>
+	      </snapshots>
+	      <id>forgerock-private-releases</id>
+	      <name>ForgeRock Private Release Repository</name>
+	      <url>http://maven.forgerock.org/repo/private-releases</url>
+	    </repository>
```

Comment:
This is the only repository that should stay (all the other repositories are irrelevant for a custom node). Can you update the URL to https please?